### DISCUSSION TITLE
This fails to work for websocket connections

### DISCUSSION THREAD

**Discussion by Sakshi-7699** [l-lin/angular-datatables#1586] [Q&A] [answered]
I have established a websocket connection to fetch data for my table.
Now when I use the given method as of httpclient module, error for dt.trigger.next() comes up.

**Comment by shanmukhateja** [comment, marked as answer]
Could you provide a link to reproducible repo via StackBlitz link or GitHub repo?

**Reply by Sakshi-7699** [reply]
Is there a way to append data to table when new entry comes on the websocket? 
I tried the destroy : true in dtOptions, this simply removes the previous entries.

**Reply by shanmukhateja** [reply]
You need [row.add](https://datatables.net/reference/api/row.add()) or [rows.add](https://datatables.net/reference/api/rows.add()) APIs to dynamically add data.  Please refer to their documentation.

I'm closing this post as it's not angular-datatables specific.
